kvdb/kvdb_sqlite.go

```diff
@@ -0,0 +1,32 @@
+//go:build kvdb_sqlite
+// +build kvdb_sqlite
+
+package kvdb
+
+import (
+	"context"
+	"time"
+
+	"github.com/btcsuite/btcwallet/walletdb"
+	"github.com/lightningnetwork/lnd/kvdb/common_sql"
+	"github.com/lightningnetwork/lnd/kvdb/sqlite"
+)
+
+const (
+	// SqliteBackend is conditionally set to false when the kvdb_sqlite
+	// build tag is not defined. This will allow testing of other database
+	// backends.
+	SqliteBackend = true
+
+	testMaxConnections = 50
+)
+
+func StartSqliteTestBackend(path, table string) (walletdb.DB, error) {
+	common_sql.Init(testMaxConnections)
+	return sqlite.NewSqliteBackend(
+		context.Background(), &sqlite.Config{
+			DBPath:  path,
```

Comment:
May make sense to eventually expose more pragama options here.

Reply:
for the time being, i have exposed the Busy-timeout value